setup.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup
-import saml2idp
 
+import saml2idp
```

Comment:
It's bad idea. It can fails in different environment. Use [bumpversion](https://github.com/peritus/bumpversion) instead.